File: net/import-assets/importing_assets_use_asset.cs
// Tip: Find more about .NET SDKs at https://kontent.ai/learn/net
using Kontent.Ai.Management;

var client = new ManagementClient(new ManagementOptions
{
    ApiKey = "<YOUR_API_KEY>",
    ProjectId = "<YOUR_ENVIRONMENT_ID>"
});
var identifier = new LanguageVariantIdentifier(Reference.ByExternalId("ext-cafe-brno"), Reference.ByCodename("en-US"));

var response = await client.UpsertLanguageVariantAsync(identifier, new LanguageVariantUpsertModel
{
    Elements = ElementBuilder.GetElementsAsDynamic(new BaseElement[]
    {
        new AssetElement
        {
            Element = Reference.ByCodename("photo"),
            Value = new[]
            {
                Reference.ByExternalId("brno-cafe-image")
            }
        }
    })
});

File: ts/import-assets/importing_assets_use_asset.ts
// Tip: Find more about JS/TS SDKs at https://kontent.ai/learn/javascript
import { ManagementClient } from '@kontent-ai/management-sdk';

const client = new ManagementClient({
  environmentId: 'KONTENT_AI_ENVIRONMENT_ID',
  apiKey: 'KONTENT_AI_MANAGEMENT_API_KEY'
});

const response = await client.upsertLanguageVariant()
  .byItemExternalId('ext-cafe-brno')
  .byLanguageCodename('en-US')
  .withData((builder) => {
    return {
      elements: [
        builder.assetElement({
          element: {
            codename: 'photo'
          },
          value: [
            {
              external_id: "brno-cafe-image"
            }
          ]
        }),
      ]
    }
  })
  .toPromise();
